src/Sulu/Bundle/PreviewBundle/Preview/Preview.php

```diff
@@ -83,15 +73,15 @@ public function stop(string $token): void
             return;
         }
 
-        $this->cache->delete($token);
+        $this->cache->deleteItem($token);
```

Comment:
interesting did the CacheItemPoolInterface supported both delete and deleteItem? or is it maybe a bug which we need tackle in 2.6? Same for contains / hasItem?

Same for the other changes?

Reply:
The method name `delete` only exists on Sulu's `PreviewCache class which then abstracted away the multiple different cache interfaces we support.